packages/fast-check/src/arbitrary/dictionary.ts

```diff
@@ -64,8 +64,27 @@ export interface DictionaryConstraints {
 export function dictionary<T>(
```

Comment:
I was initially wondering you added an overload, then I noticed that my initial version had just one parameter... So it's needed. Still reading through it but so far so good